test/e2e/tests/bridge/bridge-cross-chain-swap.spec.ts

```diff
@@ -15,6 +15,7 @@ import { getBridgeFixtures } from './bridge-test-utils';
 import { DEFAULT_FEATURE_FLAGS_RESPONSE } from './constants';
 
 describe('Bridge tests', function (this: Suite) {
+  this.timeout(160000); // This test is very long, so we need an unusually high timeout
```

Comment:
this is because after fixing the test, I can see how sometimes the spec takes longer than 80s. I've checked if there was due to unmocked API requests, but it seems it's because we are doing several actions that already take some time -- we perform 3 bridges (+1 approval) which already can take 20-25 seconds each.

https://github.com/MetaMask/metamask-extension/actions/runs/14791231298/job/41529220439?pr=32461

![Screenshot from 2025-05-02 10-40-18](https://github.com/user-attachments/assets/77cf7727-df3d-4412-a2f6-75f68972f428)